.changeset/short-panthers-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@redocly/cli": minor
+---
+
+Added the `eject` and `translations` commands for use with the new Reunite-hosted product family.
```

Comment:
I don't want to be annoying in the final hour. I think we should rename the `translations` command to `translate`. Most of our commands are in verb format:

- lint
- bundle
- eject
- preview

Why not `translate`?

Reply:
good idea!

Reply:
I thought the reason it originally wasn't `translate` is that the command doesn't translate, just generate translation stubs.

Reply:
Here's the corresponding PR for the original source: https://github.com/Redocly/redocly/pull/10425